lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -784,7 +781,9 @@ sub _failed_check {
            { type => 'submit',
              name => 'action',
             value => 'fix_tests',
-             text => $request->{_locale}->text('Save and Retry'),
+             text => $request->{_locale}->text($check->columns
+                                                ? 'Save and Retry'
+                                                : 'Cancel'),
```

Comment:
Uhm, but that will call `fix_tests`, which currently doesn't know about cancelling migrations... Shouldn't we do something about that?